### ISSUE TITLE
Taking Items From Alive NPCs Isn't Stealing

### ISSUE THREAD

**Issue by Jlobblet** [Regalis11/Barotrauma#3826]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
If an outpost NPC is incapacitated for whatever reason - but you haven't aggro'd the security guards - taking items from them doesn't count as stealing. This means you can stun them and take their items and then sell them back to the same outpost to make large amounts of money. The trick is to use a toy hammer, since it doesn't cause the guards to become 

**Steps To Reproduce**
1. Obtain a toy hammer.
2. Approach an outpost NPC.
3. Repeatedly hit them, then grab, then try to take items.
   It's tough, but with a bit of practice can be reliably accomplished.
4. Observe how no one on the outpost cares that you just mugged someone.

**Version**
0.10.4.0+

**Comment by Regalis11** [comment]
I wasn't able to fully reproduce this. As long as someone sees you stealing from the stunned/unconscious character, security seems to be alerted as expected. I did however spot one issue: no-one would react to the stealing by speaking because the game would just try to make the stunned character speak and call it a day. Fixed that one in https://github.com/Regalis11/Barotrauma-development/commit/dbeed419ebd2b5632588b4a6f73d518f4745d0a5
